notebooks/gradient_decent.ipynb

```diff
@@ -11,46 +11,32 @@
    "cell_type": "markdown",
```

Comment:
<p>Verify that changing x[:, 0] to x[:, 1] does not create any issue</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/m2lines/L96_demo/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Here is the issue @yaniyuval and @adcroft